### PR TITLE
Update landing template

### LINKED ISSUE #35: New design for landing page
Implement based on screens from Noga

### PR DESCRIPTION
**Pull request by shiryz**
Restyle the design for the landing page template.
Fixes #35 .

### CODE REVIEW

File: public/styles/_landing-page.scss [hunk 1]
```diff
@@ -1,11 +1,155 @@
...
   }
 }
+
+.goal-item-rating-progress {
+  background-color: #5DEE3D;
+  opacity: 0.3;
+  position: absolute;
+  top: 0;
+  left: 0;
+  // width: 70%;
```
Comment (des-des) [inline public/styles/_landing-page.scss]: :fire:

**Comment by des-des** [comment]
@shiryz why more commits after I am reviewing!

**Comment by des-des** [comment]
ok I have looked at those also now

File: public/styles/_landing-page.scss (hunk 1 above)
Comment (shiryz) [inline public/styles/_landing-page.scss]: Is it just the comment or something else? :crying_cat_face:

**Comment by shiryz** [comment]
@des-des , so I wouldn't give you too much work :sweat_smile:

File: public/styles/_landing-page.scss (hunk 1 above)
Comment (des-des) [inline public/styles/_landing-page.scss]: yeah just the comment

**Comment by des-des** [comment]
@shiryz I guessing this is ready to go? re-assign me!

**Pull request merged**